packages/perseus-editor/src/components/locked-figures-section.tsx

```diff
@@ -62,14 +62,23 @@ const LockedFiguresSection = (props: Props) => {
 
     return (
         <View style={styles.container}>
-            {figures?.map((figure, index) => (
-                <LockedFigureSettings
-                    key={`${uniqueId}-locked-${figure}-${index}`}
-                    {...figure}
-                    onChangeProps={(newProps) => changeProps(index, newProps)}
-                    onRemove={() => removeLockedFigure(index)}
-                />
-            ))}
+            {figures?.map((figure, index) => {
+                // TODO(LEMS-1740): Remove this check when
+                // locked lines are added to LockedFigureSettings.
+                if (figure.type === "line") {
+                    return;
+                }
```

Comment:
There's a codecov warning, but this section is going to be added + tested in the "Add a locked line" PR.